index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="main-section">
+        <header class="header">
+            <div id="logo-text">My Logo</div>
+            <div><img src="images/icon-menu.png" alt="icon-menu" id="img-1"></div>
+        </header>
+        <main class="main-content">
+            <section class="section-1">
```

Comment:
- Hey, let's also add the missing background image of this section to improve our design as seen in the Figma design below:


*your design*
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/59397159/5e179f4a-1eb0-4ee6-b9e8-f7976bf8e9de)

*Figma design*
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/59397159/831874cc-d68a-4685-b9d6-8b65f33f1f40)